### PR TITLE
 DS-3245: CSV linebreaks not supported by Bulkedit -  DSpace 6

### PR DESCRIPTION
**Pull request by rivaldi8**
Port of #1757 to dspace-6_x branch.

Fixes https://jira.duraspace.org/browse/DS-3245

### CODE REVIEW

**Review by terrywbrady — APPROVED** [review]
Tested in 6x, Tom Desair also reviewed this.  See PR 1757

File: dspace-api/src/test/java/org/dspace/app/bulkedit/DSpaceCSVTest.java [hunk 1]
```diff
@@ -43,7 +43,8 @@ public  void testDSpaceCSV()
                             "+,56599ad5-c7d2-4ac3-8354-a1f277d5a31f,Easy line,\"Lewis, Stuart\",A nice short abstract",
                             "+,56599ad5-c7d2-4ac3-8354-a1f277d5a31f,Two authors,\"Lewis, Stuart||Bloggs, Joe\",Two people wrote this item",
                             "+,56599ad5-c7d2-4ac3-8354-a1f277d5a31f,Three authors,\"Lewis, Stuart||Bloggs, Joe||Loaf, Meat\",Three people wrote this item",
-                            "+,56599ad5-c7d2-4ac3-8354-a1f277d5a31f,\"Two line\ntitle\",\"Lewis, Stuart\",abstract",
+                            "+,56599ad5-c7d2-4ac3-8354-a1f277d5a31f,\"Two line\n\ntitle\",\"Lewis, Stuart\",abstract",
```
Comment (helix84) [inline dspace-api/src/test/java/org/dspace/app/bulkedit/DSpaceCSVTest.java]: The change on this line seems unintentional; the two newlines test is added  below.

**Comment by helix84** [comment]
A late +1 apart from my comment above.

File: dspace-api/src/test/java/org/dspace/app/bulkedit/DSpaceCSVTest.java (hunk 1 above)
Comment (rivaldi8) [inline dspace-api/src/test/java/org/dspace/app/bulkedit/DSpaceCSVTest.java]: Yes, sorry I'm late, that was a mistake.

**Pull request merged**